go.mod

```diff
@@ -14,4 +14,4 @@
 
 module github.com/google/btree
 
-go 1.12
+go 1.18
```

Comment:
is it backward compatible?

Reply:
Yes, it should be.  I compiled and ran tests locally with Go 1.12 successfully.

Reply:
I've added a change to the GitHub actions to check all go versions from 1.11 through 1.18.